pkg/build/a11y/README.md

```diff
@@ -0,0 +1,23 @@
+# Pa11y accessability tests
+
+We use pa11y to run some automated simple accessability tests. They're ran with dagger to help orchestrate starting server + tests in a reproducable manner.
+
+To run the tests locally:
+
+1. Install dagger locally https://docs.dagger.io/install/
+2. Grab the grafana.tar.gz artifact by either
+   1. Downloading it from the Github Action artifact from your PR
+   1. Build it locally with:
+      ```sh
+      cd ../ # move up to the parent directory of your grafana checkout
+      dagger run go -C grafana run ./pkg/build/cmd artifacts -a targz:grafana:linux/amd64 --grafana-dir="${PWD}/grafana" > out.txt
+      cat out.txt # Will output the path to the grafana.tar.gz 
```

Comment:
We should probably not ask folks to cd upwards. `go -C x run` is the same as `cd x && go run` (i.e. the `-C` is `working-directory`).

Just drop the `-C path` flag :)